std/recursion/plonk/verifier_test.go

```diff
@@ -42,10 +42,12 @@ func (c *OuterCircuit[FR, G1El, G2El, GtEl]) Define(api frontend.API) error {
 }
 
 /*
-TODO: Tests without api.Commit fail because the current optimized MultiScalarMul (JointScalarMul in particular) requires input points to be distinct.
-
-//-----------------------------------------------------------------
+///-----------------------------------------------------------------
```

Comment:
If we have PLONK verifier option then can enable the test to serve as an example? I agree that by default should use unsafe MSM and Add in verifier for efficinecy, the edge cases happen only for very trivial inner circuits.

Reply:
Agree